akka-actor-typed/src/main/scala/akka/actor/typed/internal/adapter/GuardianStartupBehavior.scala

```diff
@@ -27,29 +19,29 @@ private[akka] object GuardianStartupBehavior {
  * system, and we know that the bootstrap is completed and the actor context can be accessed.
  */
 @InternalApi
-private[akka] final class GuardianStartupBehavior[T](val guardianBehavior: Behavior[T]) extends AbstractBehavior[Any] {
+private[akka] object GuardianStartupBehavior {
```

Comment:
can be good to review this change of GuardianStartupBehavior. Changed it to functional style since the whole purpose is that we don't have a context at startup.

Reply:
Yep, good change.